content/5_Plat/Sqrt.mdx

```diff
@@ -50,58 +50,149 @@ $$
 $$
 
 where $\texttt{blocks}[i]$ represents the total sum of the $i$-th block, the
-$i$-th block represents the sum of the elements from the range
-$[i\cdot \texttt{blocksz},(i + 1)\cdot \texttt{blocksz})$, and
-$R=\left\lfloor \frac{r}{\texttt{blocksz}} \right\rfloor$.
+$i$-th block represents the sum of the elements from the range $[i\cdot
+\texttt{blocksz},(i + 1)\cdot \texttt{blocksz})$, and $R=\left\lfloor
+\frac{r}{\texttt{blocksz}} \right\rfloor$.
 
 Finally, $\sum_{i=l}^{r} \texttt{nums}[i]$ is the difference between the two
-sums $\sum_{i=0}^{r}\texttt{nums}[i]$ and $\sum_{i=0}^{l-1}\texttt{nums}[i]$, which each are calculated in $\mathcal{O}(\sqrt N)$.
+sums $\sum_{i=0}^{r}\texttt{nums}[i]$ and $\sum_{i=0}^{l-1}\texttt{nums}[i]$,
+which each are calculated in $\mathcal{O}(\sqrt N)$.
 
 <LanguageSection>
 
 <CPPSection>
 
 ```cpp
-//CodeSnip{CPP Short Template}
+#include <bits/stdc++.h>
+using namespace std;
 
-ll blocks[450]; // 450^2 > 2e5
-int nums[(int)(2e5 + 1)];
-int block_sz;
+vector<long long> blocks;
+vector<int> nums;
+int block_size;
```

Comment:
If you are using `block_size` here instead of `block_sz` make sure to change it in the tutorial as well.